taxonomies/ingredients.txt

```diff
@@ -23992,6 +24099,7 @@ vegetarian:en:yes
 en:semolina flour
 de:Grießmehl
 es:harina de sémola
+fr:Semoule de blé
```

Comment:
```suggestion
fr:Farine de semoule
```

Reply:
actually, trigo is blé / wheat. do we split ?